src/Castle.Core/Castle.Core.csproj

```diff
@@ -44,6 +44,15 @@
 		<PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
 	</ItemGroup>
 
+	<ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+		<PackageReference Include="System.Reflection.Emit" Version="4.7.0" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)'=='netstandard2.1'">
+		<PackageReference Include="System.Configuration.ConfigurationManager" Version="4.7.0" />
+		<PackageReference Include="System.Diagnostics.EventLog" Version="4.7.0" />
```

Comment:
You already removed `FEATURE_EVENTLOG` from the conditional compilation symbols. (as per my earlier suggestion). Do we still need this dependency? If not, please remove.